### ISSUE TITLE
[ Rockbot 1 | 1.20.041 | Android | Play Store ] Intro Stage: Blue Shield enemy takes frontal damage.

### ISSUE THREAD

**Issue by ghost** [protoman/rockbot#59]
Issue #37  has not been fixed (sorry again for the erroneous app version number).

The OrangeShielder does not take damage, the blue ones do.

PS.  I report errors when I encounter them.  Don't expect me to run through the whole game from the beginning to see if it occurs everywhere or sporadically.  If I note something again during further gameplay, I won't create two issues here, but I'll mention it in the explanation - like I did before.  Would be nice if you checked that you actually fixed the problem - before you close an issue here.  At least try playing at the reported spot to see if you really fixed it.  I'm playing and reporting as a tester, I don't expect a Dev to run through the whole game as a player and check everything - that's what testers are for, that's what players should do.

![blueshielder takes frontal damage orangeshielder does not](https://user-images.githubusercontent.com/33324878/32488378-ad4f20ac-c3ac-11e7-819e-25f50723c338.gif)

**Comment by ghost** [comment]
Also happens in the final boss stage.

**Comment by protoman** [comment]
OK, let me clarify; when I close an issue, you will notice there is a link to a commit. So, it means the code was changed with the fix and I tested it. This does not mean the release version I compiled and made available before the fix, contains the fix ;)
It would be nice to have continuous integration in the project, meaning, each time I make a commit, it would make a new build for testing. But that is something a bit complex to do, even if I make my home computer run this as a server.
